compiler/build.gradle

```diff
@@ -10,9 +10,11 @@ dependencies {
   implementation project(path: ':blueprint-annotations')
   implementation 'com.google.auto.service:auto-service:1.0-rc3'
   implementation 'com.squareup:javapoet:1.9.0'
-  implementation files("${System.properties['java.home']}/../lib/tools.jar")
+//  implementation files("${System.properties['java.home']}/../lib/tools.jar")
   implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
   implementation 'androidx.recyclerview:recyclerview:1.0.0'
+  implementation 'org.apache.commons:commons-lang3:3.0'
+  runtimeOnly files('/Library/Java/JavaVirtualMachines/jdk1.8.0_311.jdk/Contents/Home/lib/tools.jar')
```

Comment:
I think this was added for test purposes? I don't think we can specify this for the repo. We might need to revert line 13

Reply:
Done